cmd/gomod2rpmdeps/gomod2rpmdeps.go

```diff
@@ -162,6 +162,12 @@ func pseudoVersionToRpmVersion(pseudoVersion string) (string, error) {
 		}
 
 		return fmt.Sprintf("%s+git%s.%s", strings.TrimPrefix(substrings[0], "v"), substrings[1], substrings[2]), nil
+		// FIXME: these are not parsed properly. '0.' and 'alpha.' are documented here https://golang.org/ref/mod#pseudo-versions
```

Comment:
can't this be part of the previous commit?

Reply:
Sure, I can squash, wasn't sure whether to make this more obvious through a separate commit, to remove, to squash.